samples/extensions/open_cl_interop/open_cl_interop.cpp

```diff
@@ -572,7 +572,7 @@ void OpenCLInterop::prepare_shared_image()
 	sampler_create_info.magFilter   = VK_FILTER_LINEAR;
 	sampler_create_info.minFilter   = VK_FILTER_LINEAR;
 	sampler_create_info.mipmapMode  = VK_SAMPLER_MIPMAP_MODE_LINEAR;
-	sampler_create_info.maxLod      = (float) 1;
```

Comment:
I'm not a C++ pro, so I was wondering why the static cast here is better? Does the old code generate warnings?

Reply:
Fixed with suggested